### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by tkimnguyen**
This is as far as I got

### CODE REVIEW

**Comment by ksuess** [comment]
@tkimnguyen you may want to continue the existing pr instead of opening a new.
https://github.com/plone/training/pull/501/files

File: plone_training_config/instructions.rst [hunk 1]
```diff
@@ -222,7 +222,10 @@ Install pre-requisites.
         # macOS
         brew install nvm
 
-        #Linux
+	# Ubuntu 20.04 LTS – see https://github.com/nvm-sh/nvm#installing-and-updating
```
Comment (jensens) [inline plone_training_config/instructions.rst:225]: Over specific, this work on other derivates with bash too.

**Comment by stevepiercy** [comment]
@tkimnguyen with the change to MyST from rST, this PR is obsolete. It may be easier to start a new PR and close this one, than to try a rebase.

**Comment by stevepiercy** [comment]
@tkimnguyen we now have two sets of instructions for setting up Plone:

- docs/plone_training_config/instructions_plone5.md
- docs/plone_training_config/instructions.md

Would you please carry these fixes over to the appropriate files, assuming they are still relevant. Thank you!

File: plone_training_config/instructions.rst (hunk 1 above)
Comment (tkimnguyen) [reply plone_training_config/instructions.rst:225]: I think here I just wanted to leave a breadcrumb so others would know where this curl command came from, and I did not try to verify that it worked on other distros

**Comment by tkimnguyen** [comment]
I'm reviewing my changes to see if I can carry them over to the new files

**Comment by tkimnguyen** [comment]
I'm applying the changes manually to https://github.com/plone/training/pull/573

File: plone_training_config/instructions.rst (hunk 1 above)
Comment (tkimnguyen) [reply plone_training_config/instructions.rst:225]: you're right... too detailed

**Comment by tkimnguyen** [comment]
closing this in favour of https://github.com/plone/training/pull/573

**Pull request closed without merging**